crates/metaslang/graph_builder/Cargo.toml

```diff
@@ -0,0 +1,45 @@
+[package]
+version.workspace = true
+rust-version.workspace = true
+edition.workspace = true
+publish = true
+
+name = "metaslang_graph_builder"
+description = "Construct graphs from parsed source code"
+homepage = "https://nomicfoundation.github.io/slang/"
+repository = "https://github.com/NomicFoundation/slang/"
+authors = [
+  "Douglas Creager <dcreager@dcreager.net>",
```

Comment:
I think we also discussed not including the original authors in this metadata, since they are not responsible for maintaining it, and shouldn't be held liable for communication/ownership of publishing its releases.
I think the LICENSE headers and files state their authorship of the original work? but we do own 100% responsibility of publishing/maintaining the crate.

Reply:
I removed all other author references, but thought we were retaining this one. We should ask legal.